apps/super-app/Makefile

```diff
@@ -0,0 +1,21 @@
+.DEFAULT_GOAL:=help
+
+##@ Dev
+init:
+	npm ci
+
+dev:
+	npm start
+
+copy-common: ## Copy common files from the personal-calls common directory
+	rm -rf src/common
+	cp -r ../../personal-calls/frontend/src/common src/
+# Do some simple transformations on the files
+	find src/common -name '*.[jt]s' | xargs -I% src/common/transform.sh %
```

Comment:
Is `transform.sh` available somewhere? I can't find it in the given directory in `personal-calls`

Reply:
Gah, that's out of date. Will fix.